src/main/scala/org/globalforestwatch/summarystats/carbon_sensitivity/CarbonSensitivityGridSources.scala

```diff
@@ -53,56 +54,56 @@ case class CarbonSensitivityGridSources(gridTile: GridTile, kwargs:  Map[String,
   val jplAGBExtent = JplAGBextent(gridTile)
   val fiaRegionsUsExtent = FiaRegionsUsExtent(gridTile)
 
-  def readWindow(window: Extent): Either[Throwable, Raster[CarbonSensitivityTile]] = {
+  def readWindow(windowKey: SpatialKey, windowLayout: LayoutDefinition): Either[Throwable, Raster[CarbonSensitivityTile]] = {
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)